src/uct/ib/rc/base/rc_ep.c

```diff
@@ -412,9 +421,9 @@ void uct_rc_txqp_purge_outstanding(uct_rc_txqp_t *txqp, ucs_status_t status,
             /* Need to release rdma_read resources taken by get operations  */
             if ((op->handler == uct_rc_ep_get_bcopy_handler) ||
                 (op->handler == uct_rc_ep_get_bcopy_handler_no_completion)) {
-                uct_rc_op_release_get_bcopy(op);
+                uct_rc_op_release_get_bcopy(op, 1);
             } else if (op->handler == uct_rc_ep_get_zcopy_completion_handler) {
-                uct_rc_op_release_get_zcopy(op);
+                uct_rc_op_release_get_zcopy(op, 1);
```

Comment:
instead of passing 'release', call uct_rc_iface_update_reads here

Reply:
there is no iface here, I'll have to obtain it differently for bcopy and zcopy (code duplication with release functions)

Reply:
can we pass iface from the calling function?